remote/remote_state_s3.go

```
@@ -201,6 +208,15 @@ func (s3Config *RemoteStateConfigS3) GetSessionRoleArn() string {
 	return s3Config.RoleArn
 }
 
+func (s3Config *RemoteStateConfigS3) GetSessionTags() map[string]string {
+	//if s3Config.AssumeRole.SessionName != "" {}
```

golangci-lint (1.62.2)

commentFormatting: put a space between `//` and comment text

(gocritic)